Makefile

```diff
@@ -19,3 +19,36 @@ all $(TASKS) config-coverage test-coverage-html:
 
 test:
 	docker run --init -u $(shell id -u):$(shell id -g) --rm -v $(PWD):$(PWD) -w $(PWD) -eTEST_LABEL=$(TEST_LABEL) -eCTEST_ARGS=$(CTEST_ARGS) -eBUILD_DIR=$(BUILD_DIR) -eGTEST_FILTER=$(GTEST_FILTER) $(CONTAINER) make -f dev-flow.mk $@
+
+# Commands to be used inside the aklite-e2e-test docker
+BIN ?= build/src/aktualizr-lite
+SOTA_DIR = /var/sota
+DEVICE_FACTORY ?= ${FACTORY}
+DEVICE_TOKEN ?= ${AUTH_TOKEN}
+DEVICE_TAG ?= main
+
+${SOTA_DIR}:
+	mkdir -p ${SOTA_DIR}/reset-apps
+	mkdir -p ${SOTA_DIR}/compose-apps
+
+register: ${SOTA_DIR}
+	DEVICE_FACTORY=${DEVICE_FACTORY} lmp-device-register -T ${DEVICE_TOKEN} --start-daemon 0 -d ${SOTA_DIR} -t ${DEVICE_TAG}
+
+unregister:
+	@rm -rf ${SOTA_DIR}/sql.db
+	@rm ${SOTA_DIR}/*.toml
+
+check:
+	${BIN} check
```

Comment:
The `make check` works but for some reason it prints the error message at the end:

```
intel-corei7-64-lmp-68
	ostree: daf970fe86f1366e11dcf311ecc13ea1c3a15a4f62afe042fec35aff10212798
	apps:
		(off) http-server -> hub.foundries.io/msul-test/http-server@sha256:7ea3532adaf189a2269b6dc165239c0b3b18d89ef3b50d3ff88cc87a8ce803c3
		(on) simple -> hub.foundries.io/msul-test/simple@sha256:ac99b967837ad1533da099b57ed9354228f82e4c61b595155b5f04361662eb5a
		(off) test-case-001 -> hub.foundries.io/msul-test/test-case-001@sha256:d60293f0fcea917bbf6013322ee817611cd3205d0e7d72f2b8a407ef4e7076f9
		(off) test-case-002 -> hub.foundries.io/msul-test/test-case-002@sha256:6835448be7813232b0a922efb64062f8080f5da84c733f926b3fa7fc5a33b682

make: *** [Makefile:42: check] Error 16

```